ocropus-rpred

```diff
@@ -133,6 +133,8 @@ def process1(arg):
     (trial,fname) = arg
     base,_ = ocrolib.allsplitext(fname)
     line = ocrolib.read_image_gray(fname)
+    if not ocrolib.isbinarized(line):
+        raise ValueError("Image %s is not binarized" % (fname))
```

Comment:
The check above checks whether the image is a rank 3 array of floats which won't work because read_image_gray already catches that case. Here we check that an array, regardless of rank, contains only 0 and 1. However you need both to check both rank and values because an RGB image that only contained e.g. pure red (1,0,0) and pure green (0,1,0) would also be considered "binarized".